content/management/configuration-api/v3.4/index.mdx

```diff
@@ -4044,7 +4045,7 @@ Informs that a chatting Customer's session fields were updated. The webhook will
 
 ### `agent_created`
 
-Informs that a new agent account was created.
+Informs that a new agent account was created. Response will contain full state of agent resource - empty fields won't be omitted.
```

Comment:
Technically, this is not a response. I'd call it `payload`.

Reply:
I'd use a present tense here as  1) the payload will always contain that, 2) we describe a sample payload represented on the side, 3) and we use the Pr. Simple in other similar cases throughout our docs :)